code/datums/components/harvester.dm

```diff
@@ -224,7 +224,8 @@
 
 		if(/datum/reagent/medicine/kelotane)
 			target.apply_damage(weapon.force*0.6, BRUTE, user.zone_selected)
-			target.fire_act(10)
```

Comment:
Just adjust this to 15 if you want to buff this, Fire_act(X) already calls adjust_fire_stacks(X) and IgniteMob. 
Xeno's have a passive firestack reduction of 1 per tick, in addition to resisting removing 3-6 fire stacks per resist, this is why it only takes them two resists to put out their fires currently. 

This PR would make it so they only have to resist once most of the time, lmao.